extensions/avro/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -24,8 +24,10 @@
 name: "Camel Avro"
 description: "Serialize and deserialize messages using Apache Avro binary data format"
 metadata:
-  unlisted: true
-  guide: "https://camel.apache.org/components/latest/avro-dataformat.html"
+  keywords:
+  - "camel"
+  - "messaging"
+  - "transformation"
+  guide: "https://camel.apache.org/components/latest/dataformats/avro-dataformat.html"
   categories:
   - "integration"
```

Comment:
We recently switched to 100% generated quarkus-extension.yaml. The generator takes the input data from runtime POM and from Camel Catalog. Could you please run `mvn -N cq:update-quarkus-metadata` from the repo root directory and check if the result makes sense?

Based on the discussion in https://groups.google.com/forum/#!topic/quarkus-dev/uc3N_cQzG1g we stopped using keywords in most cases. They are mostly redundant because code.quarkus.io searches name and description by default anyway. So keywords make sense only in rather rare cases, if the given term is both important and occurs neither in extension name nor description. In such as case the runtime POM's property `quarkus.metadata.keywords` should be used, like in https://github.com/apache/camel-quarkus/blob/master/extensions/aws-ec2/runtime/pom.xml#L34

I should document these things ASAP and sorry that I have not done that yet.

Reply:
Good to know. Thanks for sharing :+1: